<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppShell</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="AppLabel">
      <property name="text">
       <string>应用</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="7">
     <widget class="QLineEdit" name="AppText"/>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="SelectApp">
      <property name="text">
       <string>选择应用</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ArgsOptionLabel">
      <property name="text">
       <string>参数选项</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="ArgsOption"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="ArgsValueLabel">
      <property name="text">
       <string>参数值</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QLineEdit" name="ArgsValue"/>
    </item>
    <item row="1" column="6" colspan="2">
     <widget class="QPushButton" name="ArgsPassword">
      <property name="text">
       <string>加密参数值</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="ArgsAdd">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="9">
     <widget class="QTextEdit" name="LogOut"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="ReArgs">
      <property name="text">
       <string>重置参数</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>140</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>66</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QPushButton" name="Save">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QPushButton" name="StartUp">
      <property name="text">
       <string>启动</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
